js/components/widgets/Image.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @function Image
+ * @typedef {Object} Data
+ * @property {string} type
+ * @property {url} type
+ * @param {Data} data
+ * @param {string} parrentClassName
+ * @returns {string} HTML or empty
+ */
```

Comment:
Use 2 JSDoc

Why is this file in the Link-component-for-download-section branch?